rrdd/xcp_rrdd.ml

```diff
@@ -111,7 +111,7 @@ open Rrd
 let uuid_of_domid domains domid =
   try
     Rrdd_server.string_of_domain_handle
-      (List.find (fun di -> di.Xenctrl.domid = domid) domains)
+      (Lext.List.find (fun di -> di.Xenctrl.domid = domid) domains)
```

Comment:
Most of the functions that are used from `Listext.List` below are also in the standard `List`. I only spotted one instance of `filter_map` that is special. So I think that we can drop `Lext.` from almost all of the lines below.